doc/reference/storage_powerflex.md

```diff
@@ -3,11 +3,15 @@
 
 [Dell PowerFlex](https://www.dell.com/en-us/shop/powerflex/sf/powerflex) is a software-defined storage solution from [Dell Technologies](https://www.dell.com/). Among other things it offers the consumption of redundant block storage across the network.
 
-LXD offers access to PowerFlex storage clusters by making use of the NVMe/TCP transport protocol.
+LXD offers access to PowerFlex storage clusters using both transport protocols NVMe/TCP and Dell's Storage Data Client (SDC).
```

Comment:
```suggestion
LXD offers access to PowerFlex storage clusters using either NVMe/TCP or Dell's Storage Data Client (SDC) as the transport protocol.
```
We could also leave out "as the transport protocol" if that's misleading for SDC.
The current wording implies that you can use both at the same time, but you probably can't?

Reply:
Good point, you have to decide for either one of them when creating the storage pool. Fixed.